tests/unit/views/ont/OntLibraries.spec.js

```diff
@@ -59,12 +58,13 @@ describe('OntLibraries.vue', () => {
 
   describe('printerModal', () => {
     beforeEach(() => {
+      wrapper.setData({ sortDesc: false })
```

Comment:
I thought setters were deprecated, does this need to be in the mount?

Reply:
There seems to be a real problem with the bootstrap tables. Don't know if it is something to do with the way we are building the components. Don't seem to have the same problem with the apollo components. If you don't sortDesc false everything blows up.